spec/registry/gcpDataproc/facets/GcpDataprocRunFacet.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://openlineage.io/spec/facets/1-0-0/GcpDataprocRunFacet.json",
+  "$defs": {
+    "GcpDataprocRunFacet": {
+      "allOf": [
+        {
+          "$ref": "https://openlineage.io/spec/2-0-2/OpenLineage.json#/$defs/RunFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "appId": {
+              "description": "Spark application id",
+              "type": "string"
+            },
+            "appName": {
+              "description": "Spark application name",
+              "type": "string"
+            },
+            "bactchId": {
+              "description": "Filled in case of Dataproc batch job.",
+              "type": "string"
+            },
+            "batchUuid": {
+              "description": "Filled in case of Dataproc batch job",
+              "type": "string"
+            },
+            "clusterName": {
+              "description": "Dataproc cluster name",
+              "type": "string"
+            },
+            "clusterUuid ": {
+              "description": "Dataproc cluster UUID",
+              "type": "string"
+            },
+            "jobId": {
+              "description": "Spark job id",
+              "type": "string"
+            },
+            "jobUuid": {
+              "description": "Spark job uuid",
+              "type": "string"
+            },
+            "projectId": {
+              "description": "GCP project id where the job was run",
+              "type": "string"
+            },
+            "queryNodeName": {
+              "description": "Query node summarization",
+              "type": "string"
+            },
+            "sessionId": {
+              "description": "Filled in case of Dataproc Session job",
```

Comment:
Populated only for Dataproc serverless interactive sessions. The resource id of the session, used for URL generation.